R/TealAppDriver.R

```diff
@@ -257,6 +250,35 @@ TealAppDriver <- R6::R6Class( # nolint: object_name.
       available_datasets[displayed_datasets_index]
     },
     #' @description
+    #' Test if CSS elements are visible on the page via JavaScript call.
+    #' @param selector (`character(1)`) CSS selector to check visibility.
+    #' A `CSS` id will return only one element if the UI is well formed.
```

Comment:
@averissimo one last thing. In this sentence CSS is quoted where in other two previous sentences is not. Please unify. Other than that. This is ready to go

Reply:
Good catch, given that `include_css_files` uses `CSS` with backticks, I'm changing all of them to use backticks

Thanks! :bow: